### PR TITLE
add 179

### PR DESCRIPTION
**Pull request by JoannaCode**
@FreeTymeKiyan

### CODE REVIEW

File: Medium/LargestNumber_shuna.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        for(int num : nums){
+            queue.offer(num + "");
+        }
+        StringBuilder sb = new StringBuilder();
+        while(queue.size() > 0){
+            sb.append(queue.poll());
+        }
+        /**
+         * Need care the special case like [0,0], return 0;
+         */
```
Comment (FreeTymeKiyan) [inline Medium/LargestNumber_shuna.java]: javaDoc comment is not multi-line comment, please fix

File: Medium/LargestNumber_shuna.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        /**
+         * Need care the special case like [0,0], return 0;
+         */ 
+        int sLen = sb.length();
+        int i = 0 ;
+        while(i < sLen && sb.charAt(i) == '0')
+        {
+            i++;
+        }
+        if(i == sLen) return "0";
```
Comment (FreeTymeKiyan) [inline Medium/LargestNumber_shuna.java]: Seems unnecessary. If first figure is 0, then you can return 0.

**Comment by JoannaCode** [comment]
@FreeTymeKiyan  fixed!

**Comment by FreeTymeKiyan** [comment]
LGTM

**Pull request merged**